apb.yml

```diff
@@ -7,7 +7,9 @@ tags:
   - mobile-service
 metadata:
   displayName: Keycloak
-  console.openshift.io/iconClass: fa fa-users
+  imageUrl: "https://pbs.twimg.com/profile_images/702119821979344897/oAC05cEB_400x400.png"
```

Comment:
:+1: I didn't know you could do this

Reply:
I learned on the weekend, when doing the UPS change ;-)